tests/unitary/Sidechain/test_root_checkpoint.py

```diff
@@ -66,15 +70,17 @@ def test_emissions_against_other_gauge(
     # sleep until the start of the next epoch week
     chain.mine(timestamp=(tx.timestamp // WEEK + 1) * WEEK)
 
+    last_week_claimable = 0
     # 110 weeks ensures we see 2 reductions in the rate
     for i in range(1, 110):
         # checkpointing the root gauge mints all the allocated emissions for the next week
         tx = root_gauge.checkpoint()
-        emissions = root_gauge.emissions()
+        emissions = root_gauge.emissions()  # emissions this week are delayed by a week so
 
         # now we time travel one week, and get the claimable rewards from the regular gauge
         chain.mine(timestamp=(tx.timestamp // WEEK + 1) * WEEK)
         claimable = liquidity_gauge.claimable_tokens(alice, {"from": alice}).return_value
 
         # root gauge emissions should be equal to regular gauge claimable amount
-        assert math.isclose(emissions, claimable, rel_tol=1e-6)
+        assert math.isclose(emissions, last_week_claimable, rel_tol=1e-6)
+        last_week_claimable = claimable
```

Comment:
I think this can be simplified some..  the old test checkpointed and stored `emissions`, then slept, then checked `claimable` because of the pre-minting for the coming week.  But now that we're delaying a week, you can just sleep, checkpoint, claimable, compare.